contest/static/contest/js/editContest.js

```diff
@@ -20,6 +20,7 @@ SOFTWARE.
 $(document).ready(function() {
     add_attribute('id_coowner', 'class', 'searchable');
     add_attribute('id_problem', 'class', 'searchable');
+    hide('id_owner');
```

Comment:
In fact, I'm curious about whether we can generalize the ```hide``` function with class or id level.
That will be great but fine with this PR!